### PR TITLE
review of windspeed.py 

### PR DESCRIPTION
**Pull request by vincelhx**

### CODE REVIEW

**Comment by vincelhx** [comment]
The first commint concerns cases where sigma0 <=0 or sigma0 == NaN (on the left side of the image here)

nrcs_copol
before commit : 
wspd_co_before
after commit : 
wspd_co_after

In this case in the previous version, xsarsea was replacing copol by ancillary and used ancillary for dualpol inversion. Now inversion returns NaN

**Comment by vincelhx** [comment]
the second change concerns the wind direction output from inversion. The returned angle was false before these commits. Now, the return complex can be used. 

use this formula to get wind direction in meteorological convention : 
`90 - (np.angle(-np.conj(wind_xsarsea), deg=True)) + xr_dataset.ground_heading) % 360`

**Comment by vincelhx** [comment]
Lut being symetrical, i chose phi or -phi by taking the closest angle from ancillary.

**Comment by vincelhx** [comment]
i also updated documentation. make html is ok @agrouaze

**Review by agrouaze — APPROVED** [review]

**Comment by vincelhx** [comment]
added cmod7 gmf

File: docs/examples/windspeed_inversion.ipynb
Comment (agrouaze) [inline docs/examples/windspeed_inversion.ipynb:1]: Can you check this notebook is output free?

File: src/xsarsea/windspeed/other_luts.py
Comment (agrouaze) [inline src/xsarsea/windspeed/other_luts.py:1]: name of the script other_luts.py is a bit weird but you are the one that play the more with this lib.

**Review by agrouaze — CHANGES_REQUESTED** [review]
just the notebook windspeed_inversion to check .

**Review by agrouaze — APPROVED** [review]

File: docs/examples/windspeed_inversion.ipynb
Comment (vincelhx) [reply docs/examples/windspeed_inversion.ipynb:1]: done

**Pull request merged**